integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -88,21 +120,36 @@ fn test_resharding_v3() {
     }
     let (genesis, _) = genesis_builder.build();
 
-    let TestLoopEnv { mut test_loop, datas: node_datas, tempdir } = builder
+    let mut builder = builder
         .genesis(genesis)
         .epoch_config_store(epoch_config_store)
         .clients(clients)
-        .track_all_shards()
-        .build();
+        .track_all_shards();
+    if !chunk_ranges_to_drop.is_empty() {
```

Comment:
nit: Would it be ok to skip this if and just pass the empty array as an argument to the builder?